src/ControlPanel.java

```diff
@@ -0,0 +1,30 @@
+import javax.swing.*;
+import java.awt.*;
+
+    public class ControlPanel extends JTabbedPane {
+
+        ControlPanel() {
+
+            this.setBackground(Color.red);
```

Comment:
Fine for testing, but this should be changed before merging

Reply:
Done